From 1e48d50f790b652bc7142d910e3277663a7b98d5 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
Date: Thu, 5 Feb 2009 15:03:46 -0600
Subject: [PATCH] ENGR00107224: MX51: Drop STOP mode voltage to 0.9V on TO2 boards.

Dropped the STOP mode voltage.
The debug board needs rework for TO2 boards to exit STOP mode after this change.

Signed-off-by: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
---
 arch/arm/mach-mx51/mx51_3stack.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx51/mx51_3stack.c b/arch/arm/mach-mx51/mx51_3stack.c
index f79804e..5392d99 100644
--- a/arch/arm/mach-mx51/mx51_3stack.c
+++ b/arch/arm/mach-mx51/mx51_3stack.c
@@ -164,7 +164,7 @@ static void mc13892_reg_int(void)
 			printk(KERN_INFO "cannot set LP STBY voltage\n");
 	} else {
 		/* Cannot drop voltage for TO2.0 */
-		if (regulator_set_voltage(lp, 1200000))
+		if (regulator_set_voltage(lp, 900000))
 			printk(KERN_INFO "cannot set LP STBY voltage\n");
 	}
 
-- 
1.5.4.4

